sdk/nodejs/lambda/lambdaMixins.ts

```diff
@@ -281,7 +281,7 @@ export class CallbackFunction<E, R> extends LambdaFunction {
 
             if (!args.policies) {
                 // Provides wide access to "serverless" services (Dynamo, S3, etc.)
```

Comment:
This comment is not longer true.  The new policy doesn't provide either of these.  This will be a significant breaking change for users of this (in particular in AWSX and Cloud packages).